content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+ ---
+- reviewers:
+- - caesarxuchao
+- - dchen1107
```

Comment:
These front matter lines contain an error leading to [build failure](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65a983d13d5c3e00083c657a). However, it's worth noting that we should exclude these reviewer lines from this localized version.

```suggestion
```